web-console/1-homepage/search-filter.md

```diff
@@ -0,0 +1,76 @@
+# Search and Filter
+
+- The search bar is located at the very top of the web console homepage
+- This component enables users to enter a query and receive a list of results.
+
+
+![search](img/OpenShift-Next-Homepage-AllCat-Getting-Started.png)
+- The search bar is sticky at the top of the page.
+
+![search](img/search-02.png)
+- As the user scrolls, the search will remain at the top with a portion of the background image still visible.
+
+## Search Catalog
+
+![search](img/search-03.png)
+- Entering a search term brings up a list of the first five matches.
+- The matches display the catalog item icon, name and any relevant tags.
+	- Clicking on a match will trigger the order flow for that catalog item on the [overlay panel](http://openshift.github.io/openshift-origin-design/web-console/4-patterns/overlay-panel).
+
+
+#### Implementation Details
+- Catalog items will be able to be searched by name, tag, and description
+- Catalog items with a matching name should be prioritized over those with a matching tag, which are higher priority than those with a matching description.
+- Elements with more matches should be prioritized over those with fewer.
+- No results will be highlighted until the up or down arrow key is pressed.
+- Pressing the `Enter` key while no results are selected will trigger the `View all results` action (see below for details)
+- If possible and practical, the search results panel should be updated as the user enters their search term.
+
+
+## View All Results
+![search](img/search-04.png)
+- Clicking on the `View all results` link scrolls the user down to the [catalog](http://openshift.github.io/openshift-origin-design/web-console/1-homepage/catalog) on the `All Categories` tab with an active "Keyword" filter for the search term.
+
+#### Implementation Details
+- Keyword filter can only hold a single value at a time entering a new search term will replace the previous keyword filter.
+- The currently active keyword filter may also be changed by using the keyword filter box (see below for details)
```

Comment:
There are a couple options for dealing with this that I thought of. 
1. when using the main search bar, anything you type will replace any existing keyword filters, but if you use the in-context one, it'll add a new filter
2. if you want to use multiple keywords, you search for two words.

I lean towards the first option. Given that the main bar version always takes you to a fresh page, I don't think it would be too disorienting to have them behave slightly differently